### DISCUSSION TITLE
Button to clean Vram/Unload models

### DISCUSSION THREAD

**Discussion by sangoi-exe** [Mikubill/sd-webui-controlnet#1473] [Ideas]
I'm part of the unfortunate group of people with GPUs that have low Vram, so sometimes I have to close and reopen my SD to clear the Vram after creating a few maps (I'm not sure if it's real, but I prefer to create the map instead of letting it be created during inference to save Vram), or use multiple different models from ControlNet. So, it would be immensely helpful if there was a button to unload ControlNet models, similar to what Tagger and Interrogator have.

**Comment by ghost** [comment]
Actually I have the same problem too but I think this discussion would go the Auto UI 1111 github since I think the VRAM is mostly related to the Auto UI 1111.

**Comment by sub314xxl** [comment]
True. The biggest button in UI should be "Free VRAM" or "Optimize VRAM usage", but not "Generate" :)
